promscale/visualize-data/grafana.md

```diff
@@ -0,0 +1,105 @@
+# Connect Promscale and Grafana
+Grafana ships with built-in Prometheus, PostgreSQL, and other data source
+plugins that allow you to query and visualize data from a compatible database.
+To add a data source in Grafana you must be signed as a user with organization 
+administration role privileges.
+
+To connect Grafana with Promscale, install and run Grafana version 5.3 or later.
+For information about installing Grafana, see the
+[Grafana installation documentation][grafana-install].
+
+This section shows you how to connect Promscale to Prometheus, Jaeger, and
+PostgreSQL data sources in [Grafana][grafana-homepage].
+
+<procedure>
+
+## Configuring Promscale as Prometheus data source
+
+1.  Navigate to `Configuration` → `Data sources`. The data sources page lists
+    previously configured data sources for the Grafana instance.
+1.  Click `Add data source`to see a list of all supported data sources.
+1.  Type `Prometheus` in the search field and click `Select`.
+1.  Configure the data source:
+    *   In the `Name`field, type `Promscale-metrics`.
```

Comment:
```suggestion
    *   In the `Name`field, type `Promscale-PromQL`.
```

You can also query metrics with SQL. This is about PromQL and I think it's better to name it like this but I don't have strong opinions here.